.github/dependabot.yml

```
@@ -0,0 +1,6 @@
+version: 2
+updates:
+  - package-ecosystem: "maven"
+    directory: "/"
+    schedule:
+      interval: "weekly"
```

```suggestion
      interval: "weekly"
    labels:
      - "dependencies"

```

For easy tracking, we may ask the bot to put the label "dependencies" to the PR that it made.